Handbook - add article submission process (#6936)
fleetdm/fleet: handbook/digital-experience/process-for-submitting-and-publishing-articles.md

# Process for submitting and publishing articles

> This guide includes all you need to know about Fleet’s editorial process and how articles are handled between submission and publication.

Reviewer: ```suggestion
This guide includes everything you need to know about Fleet’s editorial process between submission and publication.
```

> For help with formatting your article, check out our [article formatting guide](./article-formatting-guide).

### On this page

- [Article DRIs](#article-dris)
- [Who can publish articles?](#who-can-publish-articles)
- [Communication](#communication)
- [Schedule](#schedule)
- [How to submit an article](#how-to-submit-an-article)
- [Review process](#review-process)
- [Pre-publication checklist](#pre-publication-checklist)
- [After publication](#after-publication)
- [Related pages](#related-pages)

> ## Article DRIs

Reviewer: I know "DRI" is a common term here at Fleet, but it was new to me. I didn't know what it meant. Not sure if this warrants revising since it appears throughout the handbook. Might not be worth the trouble.

Author: This is a good point. If this wasn't a header I would suggest linking "DRI" to https://fleetdm.com/handbook/people#directly-responsible-individuals.

For simplicity, though, I'm going to leave it as is. Going forward we can perhaps try to onboard the DRI concept better for new members.

| Activity | DRI |
|:---------|:----|
| Assign & schedule | Tim Kern |
| Review | Chris McGillicuddy |
| Artwork | Mike Thomas |
| Publication | Tim Kern |

## Who can publish articles?

Everyone is welcome to suggest article ideas or submit drafts. To suggest an article, open an issue in the Fleet repo using the article suggestion template (TODO), include a summary, and tag Tim Kern.

## Communication

Use [#help-content-calendar](https://fleetdm.slack.com/archives/C03PH3BBVSM) on Slack for questions and updates related to articles.

## Schedule

> Publishing occurs according to our [content calendar](https://docs.google.com/spreadsheets/d/1IzpoIfTc7cqc9g8w20fQfap3GFcrqgOaiPrpCkowcvA/edit?usp=sharing). However, that is subject to change based on the news or information that needs to be posted. For time-sensitive articles, consult with Tim Kern in [#help-content-calendar](https://fleetdm.slack.com/archives/C03PH3BBVSM) on Slack.

Reviewer: ```suggestion
Publishing occurs according to our [content calendar](https://docs.google.com/spreadsheets/d/1IzpoIfTc7cqc9g8w20fQfap3GFcrqgOaiPrpCkowcvA/edit?usp=sharing). However, publication dates are subject to change based on the news or information that needs to be posted. For time-sensitive articles, consult with Tim Kern in [#help-content-calendar](https://fleetdm.slack.com/archives/C03PH3BBVSM) on Slack.
```

## How to submit an article

1. Familiarize yourself with our [article formatting guide](./article-formatting-guide) and writing style guide (todo).

> 2. Create a Pull Request for fleetdm.com with a file of your article, formatted in Markdown (todo).

Author: ```suggestion
2. Create a pull request for fleetdm.com with a file of your article, formatted in Markdown (todo).
```

> 2. Create a Pull Request for fleetdm.com with a file of your article, formatted in Markdown (todo).

Reviewer: I don't think "Pull Request" needs to be capitalized. It isn't on GitHub https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests.

> 3. Include any relevant images or any image suggestions or requests that you may have.

Reviewer: ```suggestion
3. Include any relevant images, image suggestions, or other design requests you may have.
```

> **IMPORTANT**: Your PR is not confidential. If the information in your article is confidential, please prepare your article in a Google Doc, or create a draft PR ~1 hour before publication time to keep your article private.

Todo: Demonstrate creating an article via Loom

## Review process

> Articles are an official voice of Fleet. Therefore, we carefully review published content to make sure Fleet, the company and the product, are accurately represented.

Reviewer: ```suggestion
Articles are official Fleet communication. We carefully review published content to make sure Fleet, the company and the product, are accurately represented.
```

- Digital Experience will review your article for spelling, punctuation, and grammar, as well as for tone and brand consistency. 
- In some cases, a technical review by a member of Engineering or Security may be necessary. 
- After review, you may be asked to make revisions to your draft.

### Typical editing timelines are as follows:

| Type | Review time |
|:-----|:------------|
| Urgent posts |One business day |
| Non-urgent posts | Three business days | 
| Posts requiring a technical review | Three business days |

> > Review times for posts requiring a technical review may vary depending on the Engineering and Security team’s schedule. E.g., if a review coincides with a product release.

Reviewer: ```suggestion
> Review times for posts requiring a technical review may vary depending on the Engineering and Security teams' schedules. E.g., if a review coincides with a product release.
```

## Pre-publication checklist

- Does the article meet the original brief and keyword goals?
- Has the article been reviewed by Digital Experience for grammar edits and brand consistency?
- Does the article require a technical review?
- Do all links work correctly and open in a new window?
- Does the article contain the correct [meta tags](./article-formatting-guide.md#meta-tags)
- Has artwork been created by Digital Experience?

> - Does the article have a clear [CTA (call to action)](./article-formatting-guide.md#customizable-cta)?

Reviewer: ```suggestion
- Does the article have a clear [call to action (CTA)](./article-formatting-guide.md#customizable-cta)?
```

## After publication

After your article is published, the Growth team will schedule your post to be promoted across our social media channels.

> > Consider also promoting your article on your personal channels to help achieve a bigger impact. Depending on the article, the Growth team make a request in [#help-promote](https://fleetdm.slack.com/archives/C026W5TC10V) on Slack.

Reviewer: ```suggestion
> Consider promoting your article on your personal channels to help achieve a bigger impact. Depending on the article, the Growth team may make a request in [#help-promote](https://fleetdm.slack.com/archives/C026W5TC10V) on Slack.
```
